src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java

```diff
@@ -50,8 +50,10 @@ public void testToEntity() {
         assertEquals(expectedQuestion.getShowGiverNameTo(), actualQuestion.getShowGiverNameTo());
         assertEquals(expectedQuestion.getShowRecipientNameTo(), actualQuestion.getShowRecipientNameTo());
         assertEquals(expectedQuestion.getShowResponsesTo(), actualQuestion.getShowResponsesTo());
-        assertEquals(expectedQuestion.getCreatedAt(), actualQuestion.getCreatedAt());
-        assertEquals(expectedQuestion.getUpdatedAt(), actualQuestion.getUpdatedAt());
+        assertNotNull(expectedQuestion.getCreatedAt());
```

Comment:
I don't think this is necessary as the test is not testing the constructor of `FeedbackQuestion`.